vispy/util/logs.py

```diff
@@ -327,3 +330,14 @@ def _handle_exception(ignore_callback_errors, print_callback_errors, obj,
             else:  # == 'node':
                 logger.error("Drawing node %s repeat %s"
                              % (node, this_print))
+
+class NumPyJSONEncoder(json.JSONEncoder):
+    def default(self, obj):
+        from ..app.backends._ipynb_util import _serialize_buffer
```

Comment:
_serialize_buffer doesn't depend on them. Was your intention with those to use them for glir commands?